server/graphql/v2/enum/ActivityType.ts

```diff
@@ -21,6 +21,11 @@ export const GraphQLActivityType = new GraphQLEnumType({
   values: Activities,
 });
 
+export const GraphQLActivityClassType = new GraphQLEnumType({
+  name: 'ActivityClassType',
+  values: Classes,
```

Comment:
Now that I see this, not a fan of `Classes` as a noon, would it still be time to consider something else?

Reply:
It is, but I think we're past the point where we could do this in this PR. It will touch all the notifications settings stuff, frontend, and backend.